components/general/FeaturedWork.tsx

```diff
@@ -33,12 +33,12 @@ export default function FeaturedWork() {
     },
   ]);
 
+  let [titleName, setTitleName] = useState("global__heading-h1");
+  let [brExist, setBrExist] = useState(true);
```

Comment:
good idea, maybe using the flexbox method would be simpler?

Reply:
Actually, they are the useless codes, I want to discuss these in later stand up meeting